src/app/app.component.html

```diff
@@ -60,5 +60,23 @@ <h3>Single selection with custom clear icon</h3>
   <p>
     Selected Bank: {{bankCtrl.value?.name}}
   </p>
+
+  <h3>Single selection with option groups</h3>
+  <p>
+    <mat-form-field>
+      <mat-select [formControl]="bankGroupsCtrl" placeholder="Bank" #singleSelect>
+        <ngx-mat-select-search [formControl]="bankGroupsFilterCtrl"></ngx-mat-select-search>
```

Comment:
ngx-mat-select-search must be inside of <mat-option>

Reply:
Thanks for your comment. The example in https://github.com/bithost-gmbh/ngx-mat-select-search/blob/a1c75d75f41206f922e549cbb80234b728bdebf1/src/app/examples/04-option-groups-example/option-groups-example.component.html#L6
is already up to date